scene/3d/joint_3d.cpp

```diff
@@ -339,6 +340,10 @@ void HingeJoint3D::set_param(Param p_param, real_t p_value) {
 
 real_t HingeJoint3D::get_param(Param p_param) const {
 	ERR_FAIL_INDEX_V(p_param, PARAM_MAX, 0);
+
+	if (is_configured()) {
```

Comment:
It seems that no joint methods currently actually read data from the physics server!